### ISSUE TITLE
Tap to record

### ISSUE THREAD

**Issue by edwardcohen** [edwardcohen/recordapp#20]
There is a delay for some reason.

**Comment by alexei1118** [comment]
It is a delay for detecting long press, how many seconds do you want for it?

**Comment by edwardcohen** [comment]
don't want a delay.

**Comment by alexei1118** [comment]
We need a delay for detecting long press, if not, we cannot distinguish single tap and long press.

**Comment by edwardcohen** [comment]
no, there is no single tap. to have it autorecord, it is a double tap. we dont need a delay.

**Comment by edwardcohen** [comment]
there is no single tap. it's a double tap
